HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -187,12 +187,47 @@ def customiseForEcalTestPR22254thresholdC(process):
 
 
 
+def customizeHLTForL3OIPR(process):
+   for seedproducer in producers_by_type(process, "TSGForOI"):
```

Comment:
Please put the PR number so that we find the PR later when just having the code here to be moved into ConfDB.